### ISSUE TITLE
W5 - Submission #2

### ISSUE THREAD

**Issue by totally85** [totally85/Twitter#3]
I am ready to have my app reviewed /cc @codepathreview

**Comment by avo1** [comment]
Have you pushed your latest code? I don't see how you use API to reply a tweet, or post new one?

**Comment by totally85** [comment]
Yes I've pushed my code. I didn't know we had to actually use API to make this happen.

**Comment by avo1** [comment]
In TwitterClient.swift, you need to implement these functions, similar to how you did with retweet and favTweet.

**Comment by totally85** [comment]
Sorry. Since it said users viewing a new tweet in their timeline was optional I didn't think I had to do it. I can update it. Thanks

**Comment by totally85** [comment]
I just updated my github.
